cloudify_rest_client/deployments.py

```diff
@@ -260,6 +261,10 @@ def create(self,
                                 plugins containing source URL will
                                 be installed from source.
         :param site_name: The name of the site for the deployment.
+        :param functions_eager: Whether the get_property and get_input
```

Comment:
well, I think we don't want to specifically address get_input and get_property, but address them as static/non-runtime/deployment-creation-elevated functions because we could add more of this kind of functions in the future. And I am not sure about the flag, I think the special behavior is the runtime eval so I would think that will be something to turn on like what a user will need to do for this.

Reply:
It is using the behaviour you described when this param is at its default setting already- you have to turn off functions_eager in order to get the new (runtime evaluation) behavious.

Reply:
renamed to `runtime_only_evaluation`